template/src/routes/route-component.ts

```diff
@@ -0,0 +1,23 @@
+import { Home } from 'pages/home';
+import { About } from 'pages/about';
+import { NotFound } from 'pages/not-found';
+import { RouteName } from './utils';
+
+const RouteComponent = new Map(
+  [
+    [
+      RouteName.Home,
+      Home,
+    ],
+    [
+      RouteName.About,
+      About,
+    ],
+    [
+      RouteName.NotFound,
+      NotFound,
+    ],
+  ],
+);
```

Comment:
What I meant here was a normal JS object:

```suggestion
const RouteComponent = {
  [RouteName.Home]: Home,
  [RouteName.About]: About,
  [RouteName:NotFound]: NotFound,
};
```
It's much easier to read IMO. What do you think?